about.html

```diff
@@ -11,63 +11,101 @@
 
 </head>
 
+<header>
```

Comment:
Great job on the markup and styling!!!! Your pages are almost pixel perfect. Very Nice !!!!!!!